openmetadata-ui/src/main/resources/ui/src/components/Settings/Users/UserProfileIcon/UserProfileIcon.component.tsx

```diff
@@ -337,7 +337,7 @@ export const UserProfileIcon = () => {
               onError={handleOnImageError}
             />
           ) : (
-            <ProfilePicture name={userName} width="36" />
+            <ProfilePicture name={currentUser?.name ?? ''} width="36" />
```

Comment:
so, we can't use displayName here since that was causing us the issue here, as we are using the name to fetch the `user details` based on `name` and since previously we were using displayName as first preference what was causing us the issue.

Reply:
Thanks for the details.